flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -148,6 +148,16 @@
 	public static final String TASK_MANAGER_NETWORK_DEFAULT_IO_MODE = "taskmanager.network.defaultIOMode";
 
 	/**
+	 * Initial backoff (in ms) for partition requests, which return partition not found.
+	 */
+	public static final String TASK_MANAGER_NETWORK_REQUEST_BACKOFF_INITIAL_MS = "taskmanager.network.requestBackoffInitial";
```

Comment:
OK, I'll remove the configuration values and just have the default values hard coded. For the integration test, I can easily get the NetworkEnvironment and overwrite the values via Reflection.